src/app/beer_garden/app.py

```diff
@@ -93,39 +93,36 @@ def initialize(self):
             )
         ]
 
-        # Only want to run the MongoPruner if it would do anything
-        ttl_config = config.get("db.ttl")
-
-        tasks, run_every = db.prune_tasks(**ttl_config)
-        if run_every:
-            if ttl_config.get("multithread", False):
-                for task in tasks:
-                    self.helper_threads.append(
-                        HelperThread(
-                            db.get_pruner(),
-                            tasks=[task],
-                            run_every=timedelta(minutes=run_every),
-                            cancel_threshold=-1,
-                        )
-                    )
-                    if ttl_config.in_progress > 0:
-                        self.helper_threads.append(
-                            HelperThread(
-                                db.get_pruner(),
-                                tasks=[],
-                                run_every=timedelta(minutes=run_every),
-                                cancel_threshold=ttl_config.in_progress,
-                            )
-                        )
-            else:
-                self.helper_threads.append(
-                    HelperThread(
-                        db.get_pruner(),
-                        tasks=tasks,
-                        run_every=timedelta(minutes=run_every),
-                        cancel_threshold=ttl_config.in_progress,
-                    )
-                )
+        # Add scheduled jobs for Mongo Pruner
```

Comment:
Should do an overall check if the pruner interval is greater than 0, then do the same for each TTL to determine if it should be scheduled.

Reply:
Resolved